### DISCUSSION TITLE
Can this project be used with vscode-neovim to get tree-sitter highlighting in VS Code?

### DISCUSSION THREAD

**Discussion by ahelwer** [nvim-treesitter/nvim-treesitter#1637] [Q&A] [answered]
I use the [vscode-neovim](https://github.com/asvetliakov/vscode-neovim) extension to get basic vim functionality in VS Code. VS Code has a [well-known tree-sitter deficiency](https://github.com/Microsoft/vscode/issues/50140). Can this plugin be used with vscode-neovim as a sort of backdoor method of getting tree-sitter-based syntax highlighting into VS Code? If so, how? It doesn't seem to just work by default.

**Comment by stsewd** [comment, marked as answer]
@ahelwer so AFAIK neovim allows to be embedded in other programs by using the rpc interface. Tree-sitter makes use of the text decorations api (extmarks), so if the neovim is sending that information the plugin should be able to render the text with those colors. I think you are better asking this on the extension repo.

**Reply by ahelwer** [reply]
Asked over there https://github.com/asvetliakov/vscode-neovim/issues/715
